opm/simulators/wells/GasLiftSingleWellGeneric.cpp

```diff
@@ -1161,8 +1163,8 @@ GasLiftSingleWellGeneric::runOptimizeLoop_(bool increase)
     // if (this->debug) debugShowBhpAlqTable_();
     if (this->debug)
         debugShowAlqIncreaseDecreaseCounts_();
-    if (this->debug)
-        debugShowTargets_();
+    //if (this->debug)
+    //    debugShowTargets_();
     bool success = false; // did we succeed to increase alq?
```

Comment:
why is this commented out.

Reply:
I think I commented it out because the amount of output it generated and almost the same information was output for each iteration of the well iterations. We can have it uncommented by default? Then it is easier to just comment it out (like I did) if the output is not interesting for the current debugging session.